recipes/x265/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2.9" %}
+
+package:
+  name: x265
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/multicoreware/x265/downloads/x265_{{ version }}.tar.gz
+  md5: 693ee4ce7929a59735f441107310f0bb
+
+build:
+  skip: true  # [vc<14]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - pkgconfig              # [unix]
+    - nasm
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/hdr10plus.h      # [unix]
+    - test -f ${PREFIX}/include/x265.h           # [unix]
+    - test -f ${PREFIX}/include/x265_config.h    # [unix]
+    - test -f ${PREFIX}/lib/libx265.so           # [linux]
+    - test -f ${PREFIX}/lib/libx265.a            # [linux]
+    - test -f ${PREFIX}/lib/libhdr10plus.so      # [linux]
+    - test -f ${PREFIX}/lib/libhdr10plus.a       # [linux]
+    - test -f ${PREFIX}/lib/pkgconfig/x265.pc    # [linux]
+    - conda inspect linkages -p $PREFIX x265     # [unix]
+    - conda inspect objects -p $PREFIX x265      # [osx]
```

Comment:
Do we have a test for windows? How would we go about that?

Reply:
maybe simply a `dll` existence test?

Reply:
Well nothing was being installed on windows. So at least revisiting this was useful